### PR TITLE
Contact listing page and delete UI

### LINKED ISSUE #5: Create list contacts UI

### PR DESCRIPTION
**Pull request by saurabh-nikam**
Contact list page UI and Contact deletion UI both are added.
Fixes #5

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Menu.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+import { Search, Settings, Plus } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+
+const Menu = ({ showMenu }) => {
+  const [isSearchCollapsed, setIsSearchCollapsed] = useState(true);
+
+  return (
+    <div>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Menu.jsx]: Why is `div` required here over a React fragment?

File: app/javascript/src/components/Dashboard/Contacts/Table/TableColumn.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import React from "react";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table/TableColumn.jsx:1]: Why have you named this file as `TableColumn.jsx`?

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        />
+        <Table
+          showAlert={() =>
+            setShowDeleteAlert(showDeleteAlert => !showDeleteAlert)
+          }
+        />
+        <Alert
+          closeButton={false}
+          isOpen={showDeleteAlert}
+          message="Are you sure you want to delete this contact? These changes cannot be undone."
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx]: Why have you mentioned `These changes`? There is only one change that's happening here. That is deleting a single contact. Fix the grammar in the message.

**Review by yedhink — CHANGES_REQUESTED** [review]
@saurabh-nikam

File: app/javascript/src/components/Dashboard/Contacts/Menu.jsx (hunk 1 above)
Comment (saurabh-nikam) [reply app/javascript/src/components/Dashboard/Contacts/Menu.jsx]: `div`  is removed

File: app/javascript/src/components/Dashboard/Contacts/Table/TableColumn.jsx (hunk 2 above)
Comment (saurabh-nikam) [reply app/javascript/src/components/Dashboard/Contacts/Table/TableColumn.jsx:1]: `TableColumn` replace with `getContactTableColumnData`

File: app/javascript/src/components/Dashboard/Contacts/index.jsx (hunk 3 above)
Comment (saurabh-nikam) [reply app/javascript/src/components/Dashboard/Contacts/index.jsx]: Fixed with 

> Are you sure you want to delete this contact? This Change cannot be undone.

File: app/javascript/src/components/Dashboard/Contacts/Table/TableColumn.jsx (hunk 2 above)
Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/Table/TableColumn.jsx:1]: @saurabh-nikam https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663318389500359?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

File: app/javascript/src/components/Dashboard/Contacts/index.jsx (hunk 3 above)
Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/index.jsx]: Why is the `C` in `Change` capitialized? Also, I have been seeing a lot of grammatical mistakes in your English sentences. Moving forward, make sure to use either Grammarly or Google Docs or both combined or some other tool to make sure that your grammar is on point before writing any English sentence in your code or when you reply.

https://handbook.neetokb.com/articles/english-grammar

Comment (saurabh-nikam) [reply app/javascript/src/components/Dashboard/Contacts/index.jsx]: Ok will take of same moving forward
I will check the sentence from google docs enabled Grammarly.

**Pull request merged**